exotations/solvers/exotica_aico_solver/src/bayesian_ik_solver.cpp

```diff
@@ -339,18 +334,18 @@ double BayesianIKSolver::GetTaskCosts()
     rhat = 0;
     R.setZero();
     r.setZero();
-    for (int i = 0; i < prob_->Cost.NumTasks; i++)
+    for (int i = 0; i < prob_->cost.num_tasks; i++)
     {
-        prec = prob_->Cost.Rho(i);
+        prec = prob_->cost.rho(i);
```

Comment:
Rho changes to rho - i.e., change from upper to lower case; this is technically incorrect if it does not match our book chapter or paper.